core/chains/evm/config/v2/defaults/Arbitrum_Mainnet.toml

```diff
@@ -0,0 +1,14 @@
+ChainID = '42161'
```

Comment:
Tedious but we must be careful to remember to keep all these files updated if we change defaults e.g. here https://github.com/smartcontractkit/chainlink/pull/6847/files